src/domain/events/__tests__/EventsSearchPage.test.tsx

```diff
@@ -281,6 +282,27 @@ test('renders filter tag when category not found in pre defined list', async ()
   await screen.findByRole('button', { name: `Poista suodatin ${keywordName}` });
 });
 
+test('saves sort state to local storage', async () => {
+  advanceTo(testDate);
+  render(<EventsSearchPage />, { mocks });
+
+  // Expand the sort selector
+  const toggleButton = await screen.findByText(/ajankohtaista/i);
+  userEvent.click(toggleButton);
+
+  act(() =>
```

Comment:
`act` shouldn't be needed here. AFAIK `userEvent` should do the wrapping.

The `act` warnings are very fiddly and at times impossible to avoid. If this fixed one, then it's fine by me for you to keep it.

Reply:
Needed to wrap the event into `act` to have the `localStorage` update.